app/views/playlists.js

```diff
@@ -0,0 +1,80 @@
+import Mn from 'backbone.marionette';
+import PlaylistView from './playlist';
+import application from '../application';
+import Playlist from '../models/playlist';
+
+
+const Playlists = Mn.CompositeView.extend({
+
+    template: require('./templates/playlists'),
+
+    childViewContainer: '#playlist-list',
+
+    childView: PlaylistView,
+
+    ui: {
+        addPlaylist: '#add-playlist',
+        playlistText: '#playlist-text',
+    },
+
+    events: {
+        'click p': 'changePlaylist',
+        'click #add-playlist': 'createPlaylist',
```

Comment:
As long as you declares `ui`, use them to bind events: `'click @ui.addPlaylist': 'changePlaylist'`